sdk/storage/storage-blob/test/browser/credentialsFromIdentity.spec.ts

```diff
@@ -0,0 +1,43 @@
+import * as assert from "assert";
+import * as dotenv from "dotenv";
+import { env, getUniqueName } from "../utils";
+import { BlobServiceClient } from "../../src";
+import { TokenCredential, InteractiveBrowserCredential } from "@azure/identity";
+dotenv.config({ path: "../.env" });
+
+describe("Test Credentials from @azure/identity - Browsers Only", () => {
+  async function verifyCredential(credential: TokenCredential) {
+    const blobServiceClient = new BlobServiceClient(
+      `https://${env.ACCOUNT_NAME}.blob.core.windows.net/`,
+      credential
+    );
+
+    const access = "container";
+    const metadata = { key: "value" };
+    const { containerClient } = await blobServiceClient.createContainer(
+      getUniqueName("container"),
+      {
+        access,
+        metadata
+      }
+    );
+    const result = await containerClient.getProperties();
+    assert.deepEqual(result.blobPublicAccess, access);
+    assert.deepEqual(result.metadata, metadata);
+  }
+
+  beforeEach(async function() {
+    if (env.TEST_MODE === "record" || env.TEST_MODE === "playback") {
+      // Not exectuing these tests in the record/playback modes. Is it required ?
+      this.skip();
+    }
+  });
+
+  // ChromeHeadless won't work because user needs to authenticate in the UI
+  // In browser, Fails with `AADSTS700054: response_type 'id_token' is not enabled for the application.`
```

Comment:
Would you mind adding this to the Readme/docs?

Reply:
Though I could find it in the link, it could have been better if the docs were more clear by mentioning where the `Implicit grant section` is.. like below.

-  **Navigate to the authentication tab and select Access tokens and ID tokens under the Implicit grant section to enable implicit grant flow**
 ![image](https://user-images.githubusercontent.com/10452642/71843490-e1292400-3078-11ea-9948-2c1443a35da0.png)

Reply:
After the config is updated as above, the test failed with a different error.
After providing the email address, it failed as follows

![image](https://user-images.githubusercontent.com/10452642/71845015-46cadf80-307c-11ea-8beb-189123db2bfd.png)